Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Breeze"
 uuid = "660aa2fb-d4c8-4359-a52c-9c057bc511da"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
For what is worth, [latest release](https://github.com/NumericalEarth/Breeze.jl/releases/latest) currently is v0.1.1, not that it matters to much for Julia package manager since this isn't registered yet.

Reply:
yeah probably no point in updating this